plugins/__init__.py

```diff
@@ -44,7 +43,7 @@
 resource_states: dict[str, dict] = dict()
 
 
-class InmantaDeprecationWarning(DeprecationWarning, InmantaWarning):
+class InmantaDeprecationWarning(DeprecationWarning):
```

Comment:
I guess we can simply get rid of the class?

Reply:
true. Will remove it then